### PR TITLE
Adding to contributor list

### PR DESCRIPTION
**Pull request by kennith**
Translating document to Traditional Chinese (ZH-TW) and Simplified Chinese (ZH-CN).

### CODE REVIEW

File: src/data/_contributors.yaml [hunk 1]
```diff
@@ -1546,3 +1546,14 @@ maciejzasada:
     - author
   google: "104812516714503133849"
   email: maciej@unit9.com
+
+kennith:
```
Comment (petele) [inline src/data/_contributors.yaml]: contributors file should be sorted by last name

Comment (kennith) [reply src/data/_contributors.yaml]: @petele fixed.

**Comment by agektmr** [comment]
Hi @kennith do you have an accompanying translation as a pull request? If so, can you add a link to it? We will pull this along with that one.

**Comment by kennith** [comment]
Hi @agektmr I started the translation for Traditional Chinese and Simplified Chinese. I submitted the review requests, but I am not sure if the translation was not correct or if the translation needs to be improved since it has not been reviewed or commented. I also prefer to make sure I am on the right direction before helping to translate more documents. 

Here are the links to the review requests:
https://gitlocalize.com/repo/107/zh-TW/review/345
https://gitlocalize.com/repo/107/zh-TW/review/356
https://gitlocalize.com/repo/107/zh-TW/review/344
https://gitlocalize.com/repo/107/zh-TW/review/346
https://gitlocalize.com/repo/107/zh-TW/review/358
https://gitlocalize.com/repo/107/zh-CN/review/347

**Comment by agektmr** [comment]
@limhenry Can you review his post on GitLocalize?

**Comment by limhenry** [comment]
Everything looks OK!

@kennith i updated this: https://gitlocalize.com/repo/107/zh-TW/review/346/segments

**Comment by kennith** [comment]
@limhenry Thanks, but I think it should be 
錯過了今年的 Chrome **開發**峰**會**? 您可以在我們的YouTube播放列表觀看全部的峰**會**視頻。

and do you know how to create a pull request from GitLocalize? (Is pull request limited to moderator?)

**Comment by agektmr** [comment]
@kennith only ones with moderator privilege can create a pull request. I'll help you do that for now.

File: src/data/_contributors.yaml [hunk 2]
```diff
@@ -900,6 +900,17 @@ petelepage:
   lanyrd: petele
   description:
     en: Pete is a Developer Advocate
+ 
+ kennith:
```
Comment (petele) [inline src/data/_contributors.yaml]: There's an extra space at the beginning of ` kennith:` that is breaking the yaml processor.

**Pull request merged**